scrum.txt

```diff
@@ -0,0 +1,13 @@
+SCRUM INTRODUCTION:
+- Scrum is most widely used framework for managing software projects.
+- It is a framework rather than a methodology for developing complex products.
+- The actvities are planned and executed in time-boxed intervals known as sprints.
+- The framework works on the basis of iterative feedback loop and incremental development in time-boxed sprints for managing software/prodcut development.
+- the framework defines the roles,events,artifacts and all the underlying rules that connect them together
+- communication is given more important in scrum as information has to be passed effectively and with transparently
+- collaboration is key for the success of scrum as employees from varied background work together to build a efficient product.
+- functioning software is one key aspect which means that at any block of time, the software is in functioning state which is not in the case of waterfall methodology.
+- teams are self organized  and cross-functional and hence there is no project manager to lead and direct the teams
+- adapatation to new business developments is easier as the framework is independent of the software/product.
+
```

Comment:
Artefacts are missing ( minimum of 3-5 needed) for example, what is sprint planning, sprint backlog, daily scrum. etc

Reply:
1. I started with the introduction of git . The commands will follow in the file2 in pradeep2 branch.
2. I have done the requested modification and added more explination
3. info on scrum artefacts will follow in the coming files in pradeep3 and pradeep4 branches

Reply:
please follow semantic versioning.